dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task04215MySQLMissingConstraints.java

```diff
@@ -0,0 +1,99 @@
+package com.dotmarketing.startup.runonce;
+
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.exception.DotDataException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotmarketing.exception.DotDataException'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)